rdr_service/dao/genomics_dao.py

```diff
@@ -1847,7 +1855,7 @@ def outreach_lookup(self, pid=None, start_date=None, end_date=None):
                 # results
                 result_query = (
                     session.query(
-                        GenomicMemberReportState.participant_id,
+                        distinct(GenomicMemberReportState.participant_id),
```

Comment:
Does this work? I always thought it was query(...).distinct(). I may be wrong.

Reply:
Yes, it was giving me multiple records for participants in API before, I checked literal SQL and the alchemy method to match logic. We also use this here in the AW3 queries, https://github.com/all-of-us/raw-data-repository/blob/03d70842df2b39561d36e360f391bfc5bdacadb1/rdr_service/genomic/genomic_data.py#L37

Reply:
Cool, sounds good.

Reply:
Yea just checking to clarify, whether it's wrapped on an attribute, or placed at end of the query, the literal translation is:
`SELECT DISTINCT genomic_member_report_state.participant_id....`